pydantic/json_schema.py

```diff
@@ -368,12 +673,30 @@ def _common_set_schema(self, schema: core_schema.SetSchema | core_schema.FrozenS
         return json_schema
 
     def generator_schema(self, schema: core_schema.GeneratorSchema) -> JsonSchemaValue:
+        """
+        Returns a schema that matches a generator schema.
```

Comment:
```suggestion
        Returns a JSON schema that represents the provided GeneratorSchema.
```
I think similar tweaks are appropriate for each of the schema generation methods (basically everything after `def any_schema` above).